### PR TITLE
updating contribution information 

### PR DESCRIPTION
**Pull request by LianeHughes**

### CODE REVIEW

File: CONTRIBUTING/adding_editing_information.md [hunk 1]
```diff
@@ -59,103 +63,117 @@ git push
...
+- Go to your repository on GitHub and click "New Pull Request".
+- Alternatively, go to the main repository on GitHub ([https://github.com/ScilifelabDataCentre/pathogens-portal](https://github.com/ScilifelabDataCentre/pathogens-portal)) and click "New Pull Request", then click the text link near the top that says "compare across forks".
+- At the top of the page, you should see a pale grey box under the 'Comparing changes' text. The right-hand "head repository" drop down should show your username/fork and the branch that you worked on. Under 'base repository' you should see "ScilifelabDataCentre/pathogens-portal" and the 'base' should read 'develop'.
+- If you're happy with the list of commits and file changes shown towards the bottom of the page, click "Create pull request".
+- Write a title and description for your pull request, and then click 'Create pull request'.
+- A member of the portal team will review the pull request as quickly as possible regarding the pull request. Once approved, the changes will show on the portal.
 
-Once created, a member of the Portal team will review your changes.
-Once approved, they will be merged and published.
+## How to add a new page
```
Comment (senthil10) [inline CONTRIBUTING/adding_editing_information.md:73]: Should this be in table of contents ?

File: CONTRIBUTING/adding_editing_information.md [hunk 2]
```diff
@@ -59,103 +63,117 @@ git push
...
 
-Because the Portal is built on Hugo, it is quite easy to run a full version of the Portal on your computer and see how your changes look while doing that. See [this page for information on how to do that](https://github.com/ScilifelabDataCentre/pathogens-portal/blob/develop/CONTRIBUTING/running_a_local_copy.md).
+## Adding available data
 
-## Available datasets
+We have a database of available data/code from Sweden that can be found [in the available data section of the portal ](http://pathogens.se/datasets/all/). Multiple different types of data can be added to this database. Please note that this database is also updated approximtely monthly by the portal team.
```
Comment (senthil10) [inline CONTRIBUTING/adding_editing_information.md:79]: Can we use different word than "database", as (technical) people might think of an external database (like sql, etc)

File: CONTRIBUTING/adding_editing_information.md [hunk 3]
```diff
@@ -59,103 +63,117 @@ git push
...
 
-_Instructions coming soon._
+In order to add entries in the database, you should edit the `available_datasets.json` within the `data`folder. You need to update the date at the top of the file to the date that you are making the update. Then you can add your entry/entries. Please note that entries must include one author affiliated with a Swedish university, should pertain to COVID-19, and the data/code should be openly available or have clear instructions for how access can be made (in the event thatthe data could not be shared openly). Each entry should be in the format:
```
Comment (senthil10) [inline CONTRIBUTING/adding_editing_information.md:81]: Missing spaces between words

1. `data`folder -> `data` folder
2. thatthe data -> that the data

File: CONTRIBUTING/adding_editing_information.md [hunk 4]
```diff
@@ -174,42 +192,87 @@ We typically describe exactly what data has been shared, how it can be re-used,
...
+banner_caption: 'caption of image'
+tags: [keyword 1, keyword 2...]
+editorials_topics: [Infectious diseases] (any topics that apply, see below for information on topics)
+editorials_authors: [Your Name]
+images: [/editorials/image_name.jpg]
+---
+
+This is the text of the editorial. This is the first paragraph.
+
+This is the second paragraph of the textt. Markdown formatting should be used in the text. For example, you can make a piece of text italic by placing an asterisk at the beginning and end, *like this*. You can make a piece of text bold by placing two asterisks at the beginning and end, **like this**. You can also add a link with square brackets following round round brackets, [like this](https://example.com/data/).
```
Comment (senthil10) [inline CONTRIBUTING/adding_editing_information.md:237]: textt -> text

File: CONTRIBUTING/adding_editing_information.md [hunk 5]
```diff
@@ -174,42 +192,87 @@ We typically describe exactly what data has been shared, how it can be re-used,
...
+
+## Events
+
+The **events** page shows events related to topics around pandemic preparedness, e.g. antibiotic resistance. Users should go to the [events page](https://pathogens.se/events/) on the portal itself and fill in the form to suggest an event. The portal team will typically add the suggestion within one working day.
+
+## Funding opportunities
+
+We collect funding opportunities relevant for topics related to pandemic preparedness, e.g. COVID-19, infectious diseases, and antibiotic resistance research. Users should go to the [funding page](https://pathogens.se/funding/) on the portal itself and fill in the form to add a funding opportunity. The portal team will typically add the suggestion within one working day.
```
Comment (senthil10) [inline CONTRIBUTING/adding_editing_information.md:254]: Are you sure about one working day ? ;)

**Review by senthil10 — COMMENTED** [review]
Great work @LianeHughes 👏 Look good to me, just few minor comments.

File: CONTRIBUTING/adding_editing_information.md (hunk 2 above)
Comment (LianeHughes) [inline CONTRIBUTING/adding_editing_information.md:79]:
```suggestion
We have a table of available data/code from Sweden that can be found [in the available data section of the portal ](http://pathogens.se/datasets/all/). Multiple different types of data can be added to this database. Please note that this database is also updated approximtely monthly by the portal team.
```

File: CONTRIBUTING/adding_editing_information.md [hunk 6]
```diff
@@ -59,103 +63,117 @@ git push
...
+## Adding available data
 
-## Available datasets
+We have a table of available data/code from Sweden that can be found [in the available data section of the portal ](http://pathogens.se/datasets/all/). Multiple different types of data can be added to this database. Please note that this database is also updated approximtely monthly by the portal team.
 
-_Instructions coming soon._
+In order to add entries in the database, you should edit the `available_datasets.json` within the `data`folder. You need to update the date at the top of the file to the date that you are making the update. Then you can add your entry/entries. Please note that entries must include one author affiliated with a Swedish university, should pertain to COVID-19, and the data/code should be openly available or have clear instructions for how access can be made (in the event thatthe data could not be shared openly). Each entry should be in the format:
```
Comment (LianeHughes) [inline CONTRIBUTING/adding_editing_information.md:81]:
```suggestion
In order to add entries in the database, you should edit the `available_datasets.json` within the `data` folder. You need to update the date at the top of the file to the date that you are making the update. Then you can add your entry/entries. Please note that entries must include one author affiliated with a Swedish university, should pertain to COVID-19, and the data/code should be openly available or have clear instructions for how access can be made (in the event that the data could not be shared openly). Each entry should be in the format:
```

File: CONTRIBUTING/adding_editing_information.md (hunk 4 above)
Comment (LianeHughes) [inline CONTRIBUTING/adding_editing_information.md:237]:
```suggestion
This is the second paragraph of the text. Markdown formatting should be used in the text. For example, you can make a piece of text italic by placing an asterisk at the beginning and end, *like this*. You can make a piece of text bold by placing two asterisks at the beginning and end, **like this**. You can also add a link with square brackets following round round brackets, [like this](https://example.com/data/).
```

**Review by KaziJahurul — APPROVED** [review]
looks good.

File: CONTRIBUTING/adding_editing_information.md [hunk 7]
```diff
@@ -59,103 +63,117 @@ git push
...
 
-Because the Portal is built on Hugo, it is quite easy to run a full version of the Portal on your computer and see how your changes look while doing that. See [this page for information on how to do that](https://github.com/ScilifelabDataCentre/pathogens-portal/blob/develop/CONTRIBUTING/running_a_local_copy.md).
+## Adding available data
 
-## Available datasets
+We have a table of available data/code from Sweden that can be found [in the available data section of the portal ](http://pathogens.se/datasets/all/). Multiple different types of data can be added to this database. Please note that this database is also updated approximtely monthly by the portal team.
```
Comment (senthil10) [inline CONTRIBUTING/adding_editing_information.md:79]:
```suggestion
We have a table of available data/code from Sweden that can be found [in the available data section of the portal ](http://pathogens.se/datasets/all/). Multiple different types of data can be added to this database. Please note that this table is also updated approximately monthly by the portal team.
```

**Pull request merged**